plugins/woocommerce/src/Internal/DataStores/Orders/CustomOrdersTableController.php

```diff
@@ -604,10 +620,28 @@ private function get_hpos_setting_for_sync() {
 
 			if ( $sync_in_progress && $sync_is_pending ) {
 				$sync_message[] = sprintf(
-					// translators: %d: number of pending orders.
-					__( 'Currently syncing orders... %d pending', 'woocommerce' ),
-					$orders_pending_sync_count
+					// translators: %s: number of pending orders.
+					__( 'Currently syncing orders... %s pending', 'woocommerce' ),
+					number_format_i18n( $orders_pending_sync_count )
 				);
+
+				if ( ! $sync_enabled ) {
```

Comment:
`$sync_enabled = true` means compatibility mode is enabled. This mode takes care of both the initial sync of orders but also propagating changes immediately when orders are updated, so there's no need to start or stop anything.

The "sync now" link allows you to trigger a one-off sync without necessarily enabling compatibility mode. The "stop" button would allow you to cancel that one-off sync while it is in progress. This is particularly important when you have a ton of orders, as this can take a while and we currently don't offer a way to stop this process (unless you want to dig into A-S jobs).